installer/app/src/views/modal/cluster-delete.js.jsx

```diff
@@ -10,8 +10,11 @@ var ClusterDelete = React.createClass({
 					<header>
 						<h2>Delete cluster {this.props.clusterID}?</h2>
 					</header>
-
-					<button type="submit" style={RedBtnCSS}>Delete Cluster</button>
+					<label>
+						<div>Type the cluster ID ({this.props.clusterID}) to confirm deletion</div>
+						<input type="text" value={this.state.confirmation} onChange={this.handleConfirmation} />
```

Comment:
This needs styling, you can copy from the [`InputPrompt` component](https://github.com/flynn/flynn/blob/master/installer/app/src/views/prompt.js.jsx#L170-L172).